docs/UserGuide.md

```diff
@@ -45,6 +50,36 @@ to organize the records of their patients.
 
 ---
 
+## About Command Input
+
+### Whitespaces
+
+Due to the design of our command parser, we are able to understand your input for any number of spaces between words (even before
+the command itself). So all these commands are actually understandable and will be treated the same:
+```
+record 01/05/2021 /s coughing, fever
+record   01/05/2021  /s       coughing, fever
+record 01/05/2021  /s   coughing,       fever
+      record 01/05/2021 /s coughing, fever
+```
+
+❗ This is important: treating all these four the same means any number of spaces in between words will only be
+considered as **ONE**. Take a look at the third input. Despite there are multiple spaces between `coughing,` and `fever`,
+it will go into the database as `coughing, fever` - same as every other lines do.
+
+### Order of Arguments
+
+A word starts with `/` marks the beginning of an argument. An argument block continues until the other one is found.
```

Comment:
The phrasing here can be a little confusing for readers. Perhaps "Words or characters starting with `/` marks the beginning of an argument" would sound a little clearer.

Reply:
Thanks! It has been updated.